OpenEXR/IlmImfTest/testDeepScanLineMultipleRead.cpp

```diff
@@ -161,14 +161,14 @@ static void read_file(const char * filename)
         for(int i=0;i<width;i++)
         {
             
-            if( samplecounts[i]!= row)
+            if( samplecounts[i]!= static_cast<unsigned int>(row))
```

Comment:
maybe change the type of `i` to `unsigned it` instead?

Reply:
oh, well, it's not i that I am casting, it is row, which is passed as an int to a bunch of functions along the way, so this seemed like less changes.